__tests__/api.ts

```diff
@@ -1,42 +1,55 @@
 import { api, fetchApi } from '../src/api'
-import { mockFetch, FetchMock } from './_helper'
+import { FetchMock, serverMock, returnResponseText, returnResponseJson } from './_helper'
 
 describe('api()', () => {
-  let fetchApi: FetchMock
+  test('uses fetch() for requests to the serlo api', async () => {
+    serverMock(
+      'https://api.serlo.org/graphql',
+      returnResponseText('<api-result>')
+    )
 
-  beforeEach(() => {
-    fetchApi = mockFetch({ 'https://api.serlo.org/graphql': '<api-result>' })
-  })
-
-  test('uses fetchApi() for requests to the serlo api', async () => {
     const req = new Request('https://api.serlo.org/graphql')
-    const response = (await api(req, fetchApi.mock)) as Response
+    const response = (await api(req)) as Response
```

Comment:
You can now delete the second argument in the function `api()` since it is not needed any more for testing.